### PR TITLE
Process no compile.json situation

### LINKED ISSUE #10577: Message if qx compile out of project folder
Now if to run `qx compile` or `qx serve` from directory where is no qx config files and structure the next error is raised:

```
Error: TypeError: Cannot read properties of undefined (reading 'forEach')
    at wrapper.createMakersFromConfig (C:\Users\goldim\AppData\Roaming\npm\node_modules\@qooxdoo\framework\lib\compiler\index.js:46524:22)
    at wrapper._loadConfigAndStartMaking (C:\Users\goldim\AppData\Roaming\npm\node_modules\@qooxdoo\framework\lib\compiler\index.js:46352:48)
    at wrapper.process (C:\Users\goldim\AppData\Roaming\npm\node_modules\@qooxdoo\framework\lib\compiler\index.js:46326:20)
    at async wrapper.processCommand (C:\Users\goldim\AppData\Roaming\npm\node_modules\@qooxdoo\framework\lib\compiler\index.js:35261:21)
    at async wrapper.main (C:\Users\goldim\AppData\Roaming\npm\node_modules\@qooxdoo\framework\lib\compiler\index.js:34736:11)
```

**Describe the solution you'd like**
To catch the error and print appropriate message that `qx` command has been launched from wrong folder.

### PR DESCRIPTION
**Pull request by goldim**
Fix #10577

### CODE REVIEW

**Review by hkollmann — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by hkollmann — CHANGES_REQUESTED** [review]
seems that the tests do not run any longer.
please try 
```qx create myapp -I --type=server -v```
and check if compile.json is v´created.

**Comment by goldim** [comment]
@hkollmann Doesn't work. Seems like `create` command uses `load` method to create empty config object.

**Comment by hkollmann** [comment]
Then your changebdo not work 🥺

**Comment by johnspackman** [comment]
@goldim does this work if there is a compile.js and no compile.json?

**Comment by goldim** [comment]
@johnspackman no it doesn't. I'll try to fix it.

**Comment by hkollmann** [comment]
@goldim : any progress with this?

**Comment by goldim** [comment]
@hkollmann no, so fa. If it blocks the release, just ignore/skip this PR. Probably I may close it.

**Review by cboulanger — APPROVED** [review]
Looks good to me but I have no idea about possible side effects

**Review by hkollmann — APPROVED** [review]

**Review by derrell — CHANGES_REQUESTED** [review]

File: source/class/qx/tool/cli/commands/Compile.js [hunk 1]
```diff
@@ -537,6 +537,10 @@ Framework: v${await this.getQxVersion()} in ${await this.getQxPath()}`);
      * @return {Boolean} true if all makers succeeded
      */
     async _loadConfigAndStartMaking() {
+      if (!this.getCompilerApi().compileJsonExists() && !qx.tool.cli.Cli.getInstance().compileJsExists()) {
+        qx.tool.compiler.Console.error("Cannot find neither compile.json nor compile.js");
```
[the remark this answers is not in the document]
Comment (goldim) [reply source/class/qx/tool/cli/commands/Compile.js:541]: I would agree about can not but in code of framework there are a lot of "Cannot". I do not know if it was done on purpose or not.

[the remark this answers is not in the document]
Comment (derrell) [reply source/class/qx/tool/cli/commands/Compile.js:541]: No problem. "Cannot" is a less-used but acceptable contraction for "can not". If it's used frequently, that's fine. So only my change from `neither` to `either` should be retained.

**Review by derrell — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Pull request merged**